src/components/common/CreateNewLibraryDialog.tsx

```diff
@@ -195,12 +195,12 @@ const CreateNewLibraryDialog: React.FC<TestProps> = ({
               data-testid="cql-library-name-text-field"
               inputProps={{
                 "data-testid": "cql-library-name-text-field-input",
+                maxlength: 64,
               }}
               helperText={
                 (formik.touched["cqlLibraryName"] ||
                   focusedField === "cqlLibraryName") &&
-                (formikErrorHandler("cqlLibraryName") ||
-                  "Library name must start with an upper case letter, followed by alpha-numeric character(s) and must not contain spaces or other special characters.")
```

Comment:
@mcmcphillips could you confirm removing this would not cause a new issue?

Reply:
I misinterpreted this code's purpose, added it back